dll/win32/imm32/imm.c

```diff
@@ -1014,13 +1014,237 @@ LRESULT WINAPI ImmEscapeW(
         return 0;
 }
 
+#ifdef __REACTOS__
+
+#define ROUNDUP4(n) (((n) + 3) & ~3)
+
+/* FIXME: Move this to proper place */
+typedef struct tagCLIENTIMC {
```

Comment:
Probably next to the other structs, or inside the imm32_undoc.h that you created in another PR?

Reply:
Moved to `imm32_undoc.h`. OK.